modules/core/src/main/scala/doobie/util/put.scala

```diff
@@ -56,7 +56,7 @@ sealed abstract class Put[A](
     ) {}
 
   def unsafeSetNonNullable(ps: PreparedStatement, n: Int, a: A): Unit =
-    if (a == null) sys.error("oops, null")
+    if (a == null) sys.error("Expected non-nullable param. Use Option to describe nullable values.")
```

Comment:
Thanks nice and simple :) 
We can put `n` into the error message to provide an even friendlier error message. Since this is all under the `if` clause we won't pay the cost of constructing the string until the null case is encountered.

Reply:
done :)